starter/script.js

```diff
@@ -1 +1,126 @@
-// put code here!
+// prompt for user name
+var userName = prompt("Welcome to Azeroth! What is your name adventurer?");
+var alphabets = /^[a-zA-Z]+$/;
+
+if(userName.match(alphabets)) {
+	alert("Greetings " + userName + "!");
+	console.log("Greetings " + userName + "!");
+}
+else {
+	alert("Username has to contain only alphabets");
+	userName = prompt("Welcome to Azeroth! What is your name adventurer?");
+	alert("Greetings " + userName + "!");
+};
+
+// make array for userScore
```

Comment:
Just wondering why you made a userScore an array and count it at the end.

Would it be better it you just made userScore = 0;
and the add the score instead of pushing into it?